### DISCUSSION TITLE
How to handle an account that cannot bind a 2FA authenticator but receives a mandatory binding notification

### DISCUSSION THREAD

**Discussion by ghost** [community/community#140665] [Other Feature Feedback, Questions, & Ideas]
english:
Hello everyone who is reading this post! I am the user of the account ypshidifu1, and I am from China. Recently, I received an email from GitHub reminding me to complete the binding of the 2FA authenticator. However, I do not have a phone number outside of China, and my Android phone has been restricted by the manufacturer, preventing me from installing apps from outside the app store. Additionally, GitHub App, Google, and other apps that can complete 2FA authentication are not available in my app store. The email also stated that I need to complete the binding of the 2FA authenticator within 45 days.

Does anyone have any solutions for bypassing GitHub's 2FA authenticator binding so that I can continue using GitHub normally?

Thank you!

Chinese：
各位看到本贴的朋友们，你们好！我是账户ypshidifu1的使用者，我是中国人。近期收到GitHub的邮件，提示我完成2fa身份验证器的绑定。但是我并没有非中国地区的手机号码，并且我的手机(Android)，已经被厂商限制，无法从应用商店外安装应用。同时，应用商店内也没有GitHub App、Google等能完成2fa身份验证的app。并且邮件中说明需要在45天内，完成2fa身份验证器的绑定。
请问各位有啥方法，能够绕开GitHub 2fa身份验证器的绑定并且能正常使用GitHub

**Comment by Anit-K-Peter** [comment]
Try using a different phone to log into your 2FA-authenticated account with Google Authenticator if that device can install the app.

**Reply by ghost** [reply]
My device has been restricted by the manufacturer, and when using APK format installation files, it prompts me to find a suitable application from the app store to open them. Even if I find one, it still requires installation from the app store.

**Comment by Sanidhyafeaturist** [comment]
You might want to explore using a third-party authenticator app, such as Microsoft Authenticator or Authy, which could be available in your app store. If that doesn't work, I recommend contacting GitHub support directly to explain your situation; they may be able to provide a workaround or an extension for the 2FA requirement. Additionally, consider using a temporary phone number service to receive the verification code, but please prioritize your privacy. Lastly, if feasible, using a VPN may allow you to access different app stores. I hope these suggestions assist you in resolving the issue.

**Reply by ghost** [reply]
How can I get in touch with the technical support team? With the help of a friend, I tried emailing GitHub, but the message was blocked by the organization.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @ypshidifu1 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
